src/otx/engine/adaptive_bs/__init__.py

```diff
@@ -0,0 +1,9 @@
+"""API for adaptive batch size."""
+
+# Copyright (C) 2024 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
```

Comment:
AFAIK, the order is license - docstring for 2.x, isn't it?

Reply:
ah, I'll align it thanks

Reply:
done https://github.com/openvinotoolkit/training_extensions/pull/3378/commits/11c3e96673a65016dca411f09ab416a73eebdce6